.circleci/config.yml

```diff
@@ -12,7 +12,7 @@ jobs:
       - image: circleci/mysql:5.7.22-ram
         environment:
           MYSQL_ALLOW_EMPTY_PASSWORD: "yes"
-          MYSQL_ROOT_PASSWORD: ''
+          MYSQL_ROOT_PASSWORD: 'root'
```

Comment:
Since we're using a password now, we can probably get rid of `MYSQL_ALLOW_EMPTY_PASSWORD`